htdocs/core/tpl/card_presend.tpl.php

```diff
@@ -180,6 +180,9 @@
 	}
 	else
 	{
+		if (!empty($object->socid) && empty($object->thirdparty)) {
```

Comment:
You should add also a test to check fetch_thirdparty() method exists because it is not true for all objects.

Reply:
Thank you, I added the check as requested.